### ISSUE TITLE
Rendering problem

### ISSUE THREAD

**Issue by shuojie819** [mugiwara85/CodeblockCustomizer#6]
don't know how to say, look at the picture below

![Obsidian_Tn5PJxydzE](https://user-images.githubusercontent.com/72399630/233797634-48db3a18-dd7f-4e31-a85c-535d732e460c.gif)

**Comment by mugiwara85** [comment]
Hi! 
Just to be sure, that we are talking about the same thing. The problem is with the middle block, between the xxx-s, right? Is that codeblock between an opening and closing ``` ? Every codeblock must start and end with ```. Or could you submit a sample document?

**Comment by shuojie819** [comment]
> Hi! Just to be sure, that we are talking about the same thing. The problem is with the middle block, between the xxx-s, right? Is that codeblock between an opening and closing ? Every codeblock must start and end with . Or could you submit a sample document?`` ``` ```` ``` ``

- The problem appears in the middle inline code.
- The following is the sample content:

```md
- xxx
	- `xxxx`
- xxx
	```c++ file:main.cpp warn:6
	int main()
	{
		printf("\nHello World")
		printf("\nHello World")
		printf("\nHello World")
		printf("\nHello World")
	}
	```
```

**Comment by mugiwara85** [comment]
I released a new version 1.1.6. This should fix the problem.

**Comment by shuojie819** [comment]
> I released a new version 1.1.6. This should fix the problem.

Ok, the problem has been fixed, thanks😊
